test/js/bun/spawn/spawn.test.ts

```diff
@@ -150,6 +150,21 @@ for (let [gcTick, label] of [
         gcTick();
       });
 
+      it("success based on exit code", async () => {
```

Comment:
I was unfortunately unable to run the tests in this file.

`bun bd test 'test/js/bun/spawn/spawn.test.ts'` fails with a `Cannot find package 'detect-libc'` error.

Reply:
That worked[^1][^2], thanks!

Would be nice if that was included in the documentation for compiling/testing.

[^1]: Well, after 16 minutes of waiting for DuckDB to compile. I thought I was doing something wrong, but apparently that's a known issue?
[^2]: 5 tests seem to fail in this file even on `main`, but I'm gonna keep poking at it.

Reply:
Some of the spawn tests seem to be failing in CI

https://buildkite.com/bun/bun/builds/15568#annotation-test/js/bun/spawn/spawn.test.ts

Reply:
The fundamental issue seems to be an "Unhandled error between tests" guard.

Even waiting for all the `Promise`s to be settled before returning from the test doesn't seem to help, nor does explicitly catching the error from each one in turn: https://github.com/oven-sh/bun/pull/19266/commits/786b6904e72eadcc784e0829d4b3fe72af4924b5#diff-c35b2f1b943ffa65bbb4a7185146dcf64d13f6b2e732a14324a53943109c717dR589-R599

This one's a chin-scratcher.